client/src/OLD-App.js

```diff
@@ -0,0 +1,228 @@
+/*global chrome*/
+
+import React, { Component } from 'react';
+import moment from 'moment';
+import STATUSES from './timer-status';
+import './App.css';
+import TimerDisplay from './components/TimerDisplay';
+import TimerButton from './components/TimerButton';
+
+class App extends Component {
+  state = {
+    pomoDuration: moment.duration(6, 'seconds'),
+    shortBreakDuration: moment.duration(3, 'seconds'),
+    longBreakDuration: moment.duration(10, 'seconds'),
+    // countdownID: null,
+    // timerDisplay: moment.duration(6, 'seconds'),
+    // timerStatus: STATUSES.NOT_SET,
+    // pomoCounter: 0,
+    countdownID: chrome.extension.getBackgroundPage().timer.countdownID,
```

Comment:
I would avoid making all these getBackgroundPage() calls here. Instead you could assign those properties in the componentWillMount hook or write a method that returns those properties based off one call to getBackroungPage().